sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/ConfigurationClientJavaDocCodeSnippets.java

```diff
@@ -270,51 +271,32 @@ public void deleteSetting() {
     }
 
     /**
-     * Generates code sample for using {@link ConfigurationClient#listSettings(SettingSelector)}
+     * Generates code sample for using {@link ConfigurationClient#listSettings(SettingSelector)}  }
```

Comment:
Does this extra curly brace break the Javadoc links? Same with the others below.